src/runtime_src/core/tools/common/ReportAieShim.cpp

```diff
@@ -99,6 +99,12 @@ ReportAieShim::writeReport( const xrt_core::device* /*_pDevice*/,
       _output << fmt4("%d") % "Row" % tile.second.get<int>("row");
       if(tile.second.find("dma") != tile.second.not_found()) {
         _output << boost::format("    %s:\n") % "DMA";
+        _output << boost::format("        %s:\n") % "FIFO";
+        for(auto& node : tile.second.get_child("dma.fifo_info")) {
+          _output << fmt16("%s") % "Size" % node.second.get<std::string>("fifo_size");
+          _output << fmt16("%s") % "Count" % node.second.get<std::string>("fifo_count");
+        }

```

Comment:
Same as above.

Reply:
Look at https://gitenterprise.xilinx.com/Linux/linux-xlnx/commit/bcffe9773b49fc82c87a30f9f78921d5b4402c5a for more information

Reply:
Fixed. Thanks